Comments.Test/IntegrationTests/API/Answers/AnswersTests.cs

```diff
@@ -45,8 +45,8 @@ public async Task Get_Answer()
 			var answerText = Guid.NewGuid().ToString();
             var userId = Guid.Empty;
 
-            SetupTestDataInDB();
-            var answerId = AddAnswer(1, userId, answerText);
+			var questionId = SetupTestDataInDB();
```

Comment:
As mentioned above, it might be good to create a helper method instead of having SetupTestDataInDB return a question id